src/pkg/scan/dao/scanner/model.go

```diff
@@ -151,7 +152,11 @@ func (r *Registration) HasCapability(manifestMimeType string) bool {
 }
 
 // GetProducesMimeTypes returns produces mime types for the artifact
-func (r *Registration) GetProducesMimeTypes(mimeType string) []string {
+func (r *Registration) GetProducesMimeTypes(mimeType string, scanType string) []string {
+	// specify the mime types returned in the request
```

Comment:
I suppose this is the code for prototype, please leverage the type in the capability to revise this.